### ISSUE TITLE
make docker building script more flexible

### ISSUE THREAD

**Issue by jlapeyre** [Qiskit-Extensions/qiskit-alt#27]
The following script for building and running the docker container takes just one argument, which is the action to take

 https://github.com/Qiskit-Extensions/qiskit-alt/blob/main/docker_tests/run_dockerfile.sh

It could use another argument, the name or revision of the container, so you can build one for a different branch. Might be better to switch to a python script.

**Comment by MozammilQ** [comment]
> Might be better to switch to a python script.

I am working on the python script. :smile:

**Comment by garrison** [comment]
Is there anything left to do here beyond #30 and #36?

Related to this issue, I've been considering whether [Earthly](https://earthly.dev/) might be a good fit for testing and deploying qiskit-alt.

**Comment by jlapeyre** [comment]
No, I think there's nothing more. I don't no anything about Earthly. But if it is easy to run locally, that's attractive.
